internal/gurnel/config.go

```diff
@@ -56,7 +56,7 @@ func (c *config) getConfigDir() (string, error) {
 
 	dir, err := c.dp.getConfigDir()
 	if err != nil {
-		return "", err
+		return "", fmt.Errorf("getting home directory: %w", err)
```

Comment:
The error here is already being wrapped on line 38, so I think this should remain unchanged.